discovery-provider/src/app.py

```diff
@@ -433,6 +429,10 @@ def configure_celery(celery, test_config=None):
                 "task": "update_aggregates",
                 "schedule": timedelta(minutes=10),
             },
+            "update_aggregates": {
```

Comment:
duped name

Reply:
thank you, great catch. should be index_latest_block